spec/checkout_sdk/accounts/accounts_integration_spec.rb

```diff
@@ -76,20 +130,19 @@
         request.threshold = 1000
         request.recurrence = frequency
 
-        expect { payout_schedules_checkout_api.accounts.update_payout_schedule('ent_sdioy6bajpzxyl3utftdp7legq', CheckoutSdk::Common::Currency::USD, request) }
+        expect { @payout_schedules_sdk.accounts.update_payout_schedule('ent_sdioy6bajpzxyl3utftdp7legq', CheckoutSdk::Common::Currency::USD, request) }
           .to raise_error(CheckoutSdk::CheckoutApiException) { |e| expect(e.error_details[:error_codes].first).to eq 'company_business_registration_number_invalid' }
       end
     end
   end
-
 end
 
-def create_entity
+private def create_entity(sdk)
```

Comment:
NIT: we don't need to specify private on all functions, just define private and all the code under will be private